python/cuml/test/test_svm.py

```diff
@@ -623,3 +623,22 @@ def test_svr_skl_cmp_weighted():
     sklSVR.fit(X, y, sample_weights)
 
     compare_svr(cuSVR, sklSVR, X, y)
+
+
+@pytest.mark.parametrize('classifier', [True, False])
+@pytest.mark.parametrize('train_dtype', [np.float32, np.float64])
+@pytest.mark.parametrize('test_dtype', [np.float64, np.float32])
+def test_svm_predict_convert_dtype(train_dtype, test_dtype, classifier):
+    X, y = make_classification(random_state=0)
```

Comment:
Could we do a small test size here explicitly (similar to the others), since this is testing just the convert_dtype parameter?

Reply:
Sure. Will make it explicit (the default is only 100 rows)

Reply:
Resolved by https://github.com/rapidsai/cuml/pull/2723/commits/ddb55480362ecb66120d981621b5f0b7ceea19a5